wicket-core/src/main/java/org/apache/wicket/request/cycle/RequestCycle.java

```diff
@@ -372,9 +372,7 @@ protected IRequestHandler handleException(final Exception e)
 			 * Call out the fact that we are processing an exception in a loud way, helps to notice
 			 * them when developing even if they get wrapped or processed in a custom handler.
 			 */
-			logExtra.warn("********************************");
-			logExtra.warn("Handling the following exception", e);
-			logExtra.warn("********************************");
+			logExtra.warn("###***===--- Handling the following exception ---===***###", e);
```

Comment:
What's the problem with the asterisks in Kafka aggregators?

Reply:
Its the lines with **************************, they spam the log with no informational value

Reply:
Let's remove the special pre/suffix for good! I can foresee the next PR suggesting to remove `###***===---` because it is weird.

Reply:
i changed it to make in configurable